changelogs/5.17.md

```diff
@@ -23,3 +23,15 @@ Consider using the `mcpe-protocol` directive in `plugin.yml` as a constraint if
 ## Fixes
 - Bowl can now be used as fuel.
 - Bells always drops themselves even when using an incompatible tool.
+
+# 5.17.1
+Released 13th August 2024.
+
+## Documentation
+- Added a note about `BlockStateData::CURRENT_VERSION`.
+
+## Fixes
+- Fixed outdated `BedrockWorldData` version, this was preventing use newer worlds.
```

Comment:
Anvil rotation too

Reply:
never mind, looks like this branch doesn't have that change

Reply:
Oops, seems to it was merged just as I was writing the changelogs

Reply:
Yeah I figured. The release process really needs to be rethought.

Reply:
Solved